tools/code_format/check_format.py

```diff
@@ -905,6 +906,14 @@ def clangFormat(file_path):
     return ["clang-format rewrite error: %s" % (file_path)]
   return []
 
+# Fix for https://github.com/envoyproxy/envoy/issues/10535
+def checkProtoiValidation(file_path):
+  exclude_path = ['v1', 'v2']
+  result = PROTO_VALIDATION_STRING.search(readFile(file_path))
```

Comment:
As an aside, I think in this script we end up reading the file contents multiple times. I commented on another PR recently that if move to a class structure, it will be easier to make this kind of read result a member variable.

Reply:
As discussed this will be handled in separate PR.